packages/postgres-database/src/simcore_postgres_database/migration/versions/1c84432e5dbb_new_comp_tasks_errors_column.py

```diff
@@ -0,0 +1,27 @@
+"""new comp_tasks errors column
+
+Revision ID: 1c84432e5dbb
+Revises: 99f9b200801c
+Create Date: 2022-04-25 15:32:36.553853+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "1c84432e5dbb"
+down_revision = "99f9b200801c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("comp_tasks", sa.Column("errors", sa.JSON(), nullable=True))
```

Comment:
anything against using JSONB instead? see [here](https://www.compose.com/articles/faster-operations-with-the-jsonb-data-type-in-postgresql/) for some reference